pkg/pdutil/pd.go

```diff
@@ -244,19 +249,57 @@ func (p *PdController) getRegionCountWith(
 
 // RemoveScheduler remove pd scheduler.
 func (p *PdController) RemoveScheduler(ctx context.Context, scheduler string) error {
-	return p.removeSchedulerWith(ctx, scheduler, pdRequest)
+	return p.pauseSchedulerWith(ctx, scheduler, pdRequest)
 }
 
-func (p *PdController) removeSchedulerWith(ctx context.Context, scheduler string, delete pdHTTPRequest) (err error) {
+// TODO add an integration test to test pause scheduler succeed when pd support related api.
+// see details at https://github.com/tikv/pd/issues/3052
+func (p *PdController) pauseSchedulerWith(ctx context.Context, scheduler string, delete pdHTTPRequest) (err error) {
+	// pause this scheduler in 300 seconds
+	body, err := json.Marshal(pauseSchedulerBody{Delay: pauseTimeout})
+	if err != nil {
+		return err
+	}
+
+	// first pause this scheduler, if the first time failed. we should return the error
+	// so put first time out of for loop. and in for loop we could ignore other failed pause.
 	for _, addr := range p.addrs {
 		prefix := fmt.Sprintf("%s/%s", schedulerPrefix, scheduler)
-		_, err = delete(ctx, addr, prefix, p.cli, http.MethodDelete, nil)
+		_, err = delete(ctx, addr, prefix, p.cli, http.MethodPost, bytes.NewBuffer(body))
 		if err != nil {
 			continue
 		}
-		return nil
 	}
-	return err
+	if err != nil {
+		return err
+	}
+	log.Info("pause scheduler at beginning", zap.String("name", scheduler))
+
+	go func() {
+		tick := time.NewTicker(2 * time.Minute)
```

Comment:
```suggestion
		tick := time.NewTicker(pauseTimeout / 2)
```

or `/ 3`
or make `2 * time.Minute` a constant.